libs/types/src/auth/auth.ts

```diff
@@ -42,11 +42,48 @@ export const UserRoleSchema: z.ZodSchema<UserRole> = z.union([
   z.literal('cardless_voter'),
 ]);
 
+/**
+ * See libs/auth/src/lockout.ts for more context.
+ */
+export type NumIncorrectPinAttemptsAllowedBeforeCardLockout =
+  | 3
+  | 4
+  | 5
+  | 6
+  | 7
+  | 8
+  | 9
+  | 10;
+export const NumIncorrectPinAttemptsAllowedBeforeCardLockoutSchema = z
+  .number()
+  .min(3)
+  .max(10)
+  .step(1) as z.ZodSchema<NumIncorrectPinAttemptsAllowedBeforeCardLockout>;
+export const DEFAULT_NUM_INCORRECT_PIN_ATTEMPTS_ALLOWED_BEFORE_CARD_LOCKOUT: NumIncorrectPinAttemptsAllowedBeforeCardLockout = 5;
+
+/**
+ * See libs/auth/src/lockout.ts for more context.
+ */
+export type StartingCardLockoutDurationSeconds = 15 | 30 | 60;
+export const StartingCardLockoutDurationSecondsSchema = z.union([
+  z.number().min(15).max(15),
+  z.number().min(30).max(30),
+  z.number().min(60).max(60),
```

Comment:
Ah nice! Even where I can use `min`/`max`/`step` (i.e. where the step is consistent), I've switched to unions of literals because they allow me to get rid of the type casts

https://github.com/votingworks/vxsuite/pull/3385/commits/daa37bc8941fc2741b056b775dfb059540bd8993